code/__DEFINES/~skyrat_defines/combat.dm

```diff
@@ -108,73 +108,84 @@
-					// feels like there should be some funny brute here but iunno
-					log_combat(user, target, "suplexes", "down on the ground (knocking down both)")
-			else
-				var/datum/wound/blunt/blute_wound = affecting.get_wound_type(/datum/wound/blunt)
-				if(blute_wound && blute_wound.severity >= WOUND_SEVERITY_MODERATE)
-					//At this point we'll be doing the medical action that's not a grab maneuver
-					return
-				//Dislocation
-				. = TRUE
-				user.changeNext_move(4 SECONDS)
-				target.visible_message(span_danger("[user.name] twists [target.name]'s [affecting.name] violently!"), \
-						span_userdanger("[user.name] twists your [affecting.name] violently!"), ignored_mobs=user)
-				to_chat(user, span_danger("You start twisting [target.name]'s [affecting.name] violently!"))
-				if(do_after(user, 4 SECONDS, target))
-					target.visible_message(span_danger("[user.name] dislocates [target.name]'s [affecting.name]!"), \
-						span_userdanger("[user.name] dislocates your [affecting.name]!"), ignored_mobs=user)
-					to_chat(user, span_danger("You dislocate [target.name]'s [affecting.name]!"))
-					affecting.force_wound_upwards(/datum/wound/blunt/moderate)
-					log_combat(user, target, "dislocates", "the [affecting.name]")
+	switch(deprecise_zone(user.zone_selected))
+		if(BODY_ZONE_HEAD)
+			if(!(target.body_position == LYING_DOWN))
+				return
+			. = TRUE
+			try_headslam(user, target, affecting)
+		if(BODY_ZONE_CHEST)
+			if(istype(user.mind.martial_art, /datum/martial_art/cqc))
+			// If you know CQC, You can't suplex and instead have the ability to use the chokehold, Sorry.
+			// Sleeping people on demand is stronger anyway.
+				return
+			// Suplex!
+			. = TRUE
+			try_suplex(user, target)
+		else // Assuming we're going for a limb...
+			var/datum/wound/blunt/blute_wound = affecting.get_wound_type(/datum/wound/blunt)
+			if(blute_wound && blute_wound.severity >= WOUND_SEVERITY_MODERATE)
+				blute_wound.try_handling(user) // handle it like any other dislocation (if you're still in combat mode, you will attempt to make it worse)
+				return
+			// Dislocate!
+			. = TRUE
+			try_dislocate(user, target, affecting)
 
 	if(.)
 		user.changeNext_move(CLICK_CD_MELEE)
 	return
 
+/// Attempts to perform a suplex after SUPLEX_TIMER, causing both to be stunned. (Why spacemen are able to do such a thing on reflex, nobody knows.)
+/datum/species/proc/try_suplex(mob/living/carbon/human/user, mob/living/carbon/human/target)
+	target.visible_message(span_danger("[user.name] holds [target.name] tight and starts lifting [target.p_them()] up!"), \
+			span_userdanger("[user.name] holds you tight and lifts you up!"), ignored_mobs=user)
+	to_chat(user, span_danger("You hold [target.name] tight and lift [target.p_them()] up..."))
+	user.changeNext_move(SUPLEX_TIMER)
+	if(!do_after(user, SUPLEX_TIMER, target) || !grab_maneuver_state_check(user, target))
+		return
+	var/move_dir = get_dir(target, user)
+	var/moved_turf = get_turf(target)
```

Comment:
Is it possible for you to type this variable as a turf?

Reply:
maybe not the dir one but the moved_turf, probably?